docs/_templates/searchbox.html

```diff
@@ -0,0 +1,6 @@
+<div role="search">
+  <form id="rtd-search-form" class="wy-form" action="{{ pathto('search') }}" method="get">
+    <input type="text" name="q" placeholder="{{ _('Search docs') }}" />
```

Comment:
```suggestion
    <input type="search" name="q" placeholder="{{ _('Search docs') }}" />
```

In addition I think this template would warrant a comment to explain what we’re overriding from the built-in template.

Reply:
This section came from @chosak

Reply:
@tbrlpld could you compare this with the built-in template and check what the differences might be? If there are none, we might as well keep the built-in template.

Reply:
This is the original from the current production site
```html
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs">
    <input type="hidden" name="check_keywords" value="yes">
    <input type="hidden" name="area" value="default">
  </form>
</div>
```

It looks like this is where the `<div role=search>` is coming from.

Reply:
I guess we are removing the two `hidden` inputs and add a `submit` input.

Reply:
Original template: https://github.com/readthedocs/sphinx_rtd_theme/blob/master/sphinx_rtd_theme/searchbox.html